fire/cli/ts/statistik_ts.py

```diff
@@ -0,0 +1,135 @@
+from dataclasses import dataclass, fields, asdict
+from datetime import datetime
+
+import numpy as np
+
+from fire.api.model import (
+    Tidsserie,
+    GNSSTidsserie,
+    HøjdeTidsserie,
+    Punkt,
+)
+
+
+@dataclass
+class Statistik:
+    TidsserieID: str
+    Ident: str
+    N: int
+    dof: int
+    ddof: int
+    grad: int
+    R2: float
+    var_0: float
+    std_0: float
+    hældning: float
+    var_hældning: float
+    std_hældning: float
+    ki_hældning_nedre: float
+    ki_hældning_øvre: float
+    mex: float
+    mey: float
+
+    def __str__(self):
+        header = ", ".join([str(field.name) for field in fields(self)])
+        linje = ", ".join([str(getattr(self, field.name)) for field in fields(self)])
+        return f"{header}\n{linje}"
```

Comment:
Det kunne godt ligne at det her faktisk burde være en `__repr__()`, men måske er der en årsag til at du har valgt `__str__()`?

Reply:
Som jeg forstår det her https://realpython.com/python-repr-vs-str/  hvor de skriver lidt nede i artiklen: 

> In Short: Use .__repr__() for Programmers vs .__str__() for Users

så synes jeg det skal være __str__. Det er meningen at man skal kunne printe indholdet af Statistik-klassen så brugeren få det ud som csv. Men hvis du synes det skal være repr så laver jeg det bare sådan. :)

Reply:
Du har ret. Det var vist mest af alt uklart for mig i hvilken sammenhæng det blev brugt.